db/testing.py

```diff
@@ -77,3 +78,22 @@ def load_low_level_data(self, mbid):
         """
         with open(self.data_filename(mbid)) as json_file:
             db.data.submit_low_level_data(mbid, json.loads(json_file.read()), gid_types.GID_TYPE_MBID)
+
+    def load_fake_low_level_data(self, mbid):
```

Comment:
I changed the name and added a docstring

The random() allows me to submit several times the same mbid as a new separate entry, to test counts > 1. I tried to make that clear in the docstring, I'm not sure whether I managed.